distribution/docs/src/main/resources/_contents/_running/running-intro-contents.adoc

```diff
@@ -0,0 +1,210 @@
+wrapper.java.additional.14=-Dfile.encoding=UTF8
+wrapper.java.additional.15=-D${ddf-branding-lowercase}.home=%${branding}_HOME%
+
+#Update the following:
+wrapper.java.maxmemory=2048
+----
++
+. Set the `${branding}_HOME` property.
++
+.INSTALLATION_HOME/etc/${branding-lowercase}-wrapper.conf
+----
+set.default.${branding}_HOME="%KARAF_HOME%"
+----
++
+. Install the wrapper startup/shutdown scripts.
++
+*Windows*
++
+Run the following command in a console window. The command must be run with elevated permissions.
++
+----
+INSTALLATION_HOME/bin/${branding-lowercase}-service.bat install
+----
+Startup and shutdown settings can then be managed through *Services MMC Start → Control Panel → Administrative Tools → Services*.
++
+*Redhat*
++
+----
+root${at-symbol}localhost# ln -s ${branding}_INSTALL/bin/${branding-lowercase}-service /etc/init.d/
+root${at-symbol}localhost# chkconfig ${ddf-branding-lowercase}-service --add
+root${at-symbol}localhost# chkconfig ${ddf-branding-lowercase}-service on
+----
++
+*Ubuntu*
++
+----
+root${at-symbol}localhost# ln -s ${branding}_INSTALL/bin/${branding-lowercase}-service /etc/init.d/
+root${at-symbol}localhost# update-rc.d -f ${ddf-branding-lowercase}-service defaults
+----
++
+*Solaris*
++
+----
+root${at-symbol}localhost# ln -s ${branding}_INSTALL/bin/${branding-lowercase}-service /etc/init.d/
+root${at-symbol}localhost# ln -s /etc/init.d/${branding-lowercase}-service /etc/rc0.d/K20${ddf-branding-lowercase}-service
+root${at-symbol}localhost# ln -s /etc/init.d/${branding-lowercase}-service /etc/rc1.d/K20${ddf-branding-lowercase}-service
+root${at-symbol}localhost# ln -s /etc/init.d/${branding-lowercase}-service /etc/rc2.d/K20${ddf-branding-lowercase}-service
+root${at-symbol}localhost# ln -s /etc/init.d/${branding-lowercase}-service /etc/rc3.d/S20${ddf-branding-lowercase}-service
+----
++
+[WARNING]
+====
+While it is not a necessary step, information on how to convert the System V init scripts to the Solaris System Management Facility can be found at http://www.oracle.com/technetwork/articles/servers-storage-admin/scripts-to-smf-1641705.html
+====
++
+.Solaris-Specific Modification
+[WARNING]
+====
+Due to a slight difference between the Linux and Solaris implementation of the `ps` command, the `${ddf-branding-lowercase}-service` script needs to be modified.
+====
++
+. Locate the following line in ${branding}_INSTALL/bin/${branding-lowercase}-service
++
+.Solaris ${branding}_INSTALL/bin/${branding-lowercase}-service
+----
+pidtest=`$PSEXE -p $pid -o command | grep $WRAPPER_CMD | tail -1`
+----
++
+. Change the word command to comm.
++
+.Solaris ${branding}_Install/bin/${branding-lowercase}-service
+----
+pidtest=`$PSEXE -p $pid -o comm | grep $WRAPPER_CMD | tail -1`
+----
+
+===== Karaf Documentation
+
+Because ${branding} is built on Apache Karaf, more information on operating ${branding} can be found in the http://karaf.apache.org/index/documentation.html[Karaf documentation].
```

Comment:
This link no longer works. I'm thinking we should just remove it rather than maintain it.